addons/mail/static/src/components/welcome_view/welcome_view.xml

```diff
@@ -2,12 +2,19 @@
 <templates xml:space="preserve">
 
     <t t-name="mail.WelcomeView" owl="1">
-        <div class="o_WelcomeView h-100 d-flex flex-column justify-content-center align-items-center bg-light">
-            <h1 class="font-weight-light">You've been invited to a meeting!</h1>
-            <h2 class="m-5" t-esc="messaging.companyName"/>
-            <div class="d-flex justify-content-center">
-                <MediaPreview class="mr-5" localId="welcomeView.mediaPreview.localId"/>
-                <div class="d-flex flex-column justify-content-center">
+        <div class="o_WelcomeView d-flex flex-column align-items-center bg-light" t-att-class="{ 'justify-content-center': !messaging.device.isMobile }">
+            <t t-if="!messaging.device.isMobile">
+                <h1 class="font-weight-light">You've been invited to a meeting!</h1>
```

Comment:
When the window is really small (or the text gets longer with translation) this (and other elements) should take some margin to avoid being too close to border.